data/changelog/odoc/2019-03-08-odoc-1.4.0.md

```diff
@@ -105,10 +105,10 @@ While release 1.4.0 has new features, it is mainly focused on stability and main
 
 In summary, compared with 1.2.0, odoc now has:
 
-- [New appearance](http://odis.io/odoc-demo/order/Order/index.html).
+- [New appearance](http://odis.io/odoc-demo/order/Order/index.html - [404 Not Found]).
```

Comment:
```suggestion
- [New appearance](https://ocaml.github.io/odoc/#overview).
```
Site no longer exists. Perhaps a link to the `odoc` website?

Reply:
A link is not relevant here. Best remove.

```suggestion
- A new appearance of generated documentation.
```